lib/main.dart

```diff
@@ -1,125 +1,79 @@
 import 'package:flutter/material.dart';
 
-void main() {
-  runApp(const MyApp());
-}
+class MyAppBar extends StatelessWidget {
+  const MyAppBar({required this.title, super.key});
 
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+  // Fields in a Widget subclass are always marked "final".
+
```

Comment:
use appbar for title you want to display on your app

Reply:
I have already used Material MyAppBar on line 51

Reply:
Make the title centered to look more appealing by setting centerTitle: true.

Reply:
I have used Expanded to use all extra space between title and search icon to put search icon  at end of right corner